vpr/src/place/initial_placement.cpp

```diff
@@ -754,7 +957,8 @@ static void clear_block_type_grid_locs(std::unordered_set<int> unplaced_blk_type
 
 bool place_one_block(const ClusterBlockId& blk_id,
                      enum e_pad_loc_type pad_loc_type,
-                     std::vector<t_grid_empty_locs_block_type>* blk_types_empty_locs_in_grid) {
+                     std::vector<t_grid_empty_locs_block_type>* blk_types_empty_locs_in_grid,
```

Comment:
Probably cleaner to pass by reference using & instead.

Reply:
Especially because the lower level helper functions are passing by reference with &

Reply:
Can't pass NULL when it is passed by reference.